### ISSUE TITLE
Different styles in similar buttons in commands bar

### ISSUE THREAD

**Issue by govindgoel** [publiclab/PublicLab.Editor#409]
What did you expect to see that you didn't?
->The ok and cancel button have different UI so this can be changed.

![ ](https://media.giphy.com/media/RM4bTJlyVAfKiJIqah/giphy.gif)

### Please show us where to look

https://publiclab.org/post
****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by Kirti2502** [comment]
can i take up this issue?

**Comment by govindgoel** [comment]
@VladimirMikulic  Can you approve this issue

**Comment by VladimirMikulic** [comment]
@govindgoel We could make pup-up modals more consistent. :+1:

**Comment by nstjean** [comment]
Good catch! We definitely want them consistent!

**Comment by NitinBhasneria** [comment]
@govindgoel @nstjean @VladimirMikulic I want to work on this issue.

**Comment by VladimirMikulic** [comment]
Go ahead 👍

**Comment by NitinBhasneria** [comment]
@VladimirMikulic I am unable to find the code. Can you help me with the code

**Comment by VladimirMikulic** [comment]
@NitinBhasneria there is no code to find, at least in our codebase. It's Woofmark's CSS in node_modules. You need to override it.

**Comment by NitinBhasneria** [comment]
@VladimirMikulic I have gone through the problem. The problem is that the insert link pop up "cancel" and "ok" button do not have classes "btn btn-default" included.

**Comment by VladimirMikulic** [comment]
Well done. You can inject them with JS.

**Comment by nicoleiocana** [comment]
@VladimirMikulic This is incorrect. The classes are to be updated in the `dist/PublicLab.Editor.js` file. Specifically, on lines 19788 & 19789 respectively. The update that upholds the DRY principal is to update the code to this:

```
    cancel: e('button', 'wk-prompt-cancel btn btn-default', 'Cancel'),
    ok: e('button', 'wk-prompt-ok btn btn-default', 'Ok'),
```

No need to inject them with js.

**Comment by VladimirMikulic** [comment]
@nicoleiocana We don't modify the dist file directly!

**Comment by nicoleiocana** [comment]
You are correct. It was bad for me to assume that the contributors understood where the files existed in the code. I admit my mistake.

We edit the file here:

`/PublicLab.Editor/node_modules/woofmark/src/prompts/render.js` lines 30 & 31.

**Comment by VladimirMikulic** [comment]
@nicoleiocana Unfortunately, you are wrong again. We never modify dist files and files in the node_modules.

**Comment by nicoleiocana** [comment]
Thanks for helping me understand. So, if can understand correctly, to respect the integrity of Woofmark's original code, any update that we make to that code can only come updates made in new modules that we create?

**Comment by VladimirMikulic** [comment]
@nicoleiocana it's not just specific to Woofmark, but really to any NPM package. The reason we don't modify files in node_modules is that every time we install/update the package again, the changes are erased!

**Comment by nicoleiocana** [comment]
Thank you for your patience and clarification. I'm making an issue right now & want to contribute to PublicLab.Editor. This solidifies a lot of the confusion that I had with working in this repo.

**Comment by VladimirMikulic** [comment]
To be honest, the dist file shouldn't be in the repo at all. It's highly discouraging and anti-pattern, but for some reason, it is present and you'll see it in a lot of PL repos. 
Don't worry though, you are in right hands since I came here, I enforce best practises as much as I can :)

**Comment by nicoleiocana** [comment]
So glad I can soak up all of the schooling that you are providing me 😂.

Can you offer your insight? Is this why the plots2 cloned repo environment looks like this:
![plots2-cloned-environment](https://user-images.githubusercontent.com/44947175/78913589-178e8a80-7a3e-11ea-9195-52fd4a194b07.png)

versus the production environment:
![plots2-production-environment](https://user-images.githubusercontent.com/44947175/78913627-24ab7980-7a3e-11ea-8f24-a699a2782633.png)

**Comment by VladimirMikulic** [comment]
@nicoleiocana yes, it's a bit tricky for newcomers. `plots2` doesn't include of all files of the public lab's website. Some of them are bundled in production only and not present in the repo (development) like the homepage for example or this editor styles that you've provided here.

**Comment by nicoleiocana** [comment]
🤔 So, if I were to develop and test here in the PublicLab.Editor repo... how am I certain that the changes will be present in the Publiclab.org site? I can see that the "last four buttons" issue has been attempted by many contributors, too many to link.

**Comment by VladimirMikulic** [comment]
You simply develop in the browser!
